hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/upgrade/OMUpgradeFinalizer.java

```diff
@@ -48,70 +39,18 @@ public OMUpgradeFinalizer(OMLayoutVersionManager versionManager) {
   @Override
   public StatusAndMessages finalize(String upgradeClientID, OzoneManager om)
       throws IOException {
-    ozoneManager = om;
-    StatusAndMessages response = preFinalize(upgradeClientID, om);
-    if (response.status() != FINALIZATION_REQUIRED) {
-      return response;
-    }
-    // This requires some more investigation on how to do it properly while
-    // requests are on the fly, and post finalize features one by one.
-    // Until that is done, monitoring is not really doing anything meaningful
-    // but this is a tradoff we can take for the first iteration either if
-    // needed, as the finalization of the first few features should not take
-    // that long. Follow up JIRA is in HDDS-4286
-    //    String threadName = "OzoneManager-Upgrade-Finalizer";
-    //    ExecutorService executor =
-    //        Executors.newSingleThreadExecutor(r -> new Thread(threadName));
-    //    executor.submit(new Worker(om));
-    try {
-      getFinalizationExecutor().execute(ozoneManager.getOmStorage(),
-          this);
-    } catch (Exception e) {
-      e.printStackTrace();
-      throw (IOException) e;
-    }
-    return STARTING_MSG;
+    return super.finalize(upgradeClientID, om);
```

Comment:
after refactoring we don't need this method as it is just delegating to super.

Reply:
Fixed